qiskit_experiments/library/randomized_benchmarking/interleaved_rb_experiment.py

```diff
@@ -121,6 +121,8 @@ def __init__(
             delay_ops = [delay.operation for delay in interleaved_element.get_instructions("delay")]
         if delay_ops:
             timing = BackendTiming(backend)
+        else:
+            timing = None
```

Comment:
Yuck, this makes pylint happy but doesn't look any better to me (since the only use of `timing` below fails if it is `None`).